documentation/src/docs/getting-started.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Getting Started with Spark
+sidebar_label: Getting Started
+---
+
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+import TOCInline from '@theme/TOCInline';
+import { IfHaveFeature, IfMissingFeature } from 'nautilus-docs';
+
+<TOCInline toc={toc} />
+
+You can run Apache Spark applications written in Java, Scala, or Python. Using Python is the easiest to get started.
+
+## Prepare Development Environment
+
+<IfHaveFeature feature="nautilus">
+
+:::tip Streaming Data Platform
+SDP users can optionally skip this entire *Prepare Development Environment* section. However, it is often useful to develop applications locally in a sandbox environment before deploying to a production system such as SDP. For this reason, we recommend that this section is not skipped.
+:::
+
+</IfHaveFeature>
+
+### Install Operating System
+
+Install Ubuntu 20.04 LTS. Other operating systems can also be used but the commands below have only been tested
+on this version.
+
+### Install Java 11
+
+```shell
+sudo apt-get install openjdk-11-jdk
+```
+
+You may have multiple versions of Java installed. Ensure that Java 11 is the default with the command below.
+
+```shell
+sudo update-alternatives --config java
+```
+
+### Run Pravega
+
+This will run a development instance of Pravega locally. The transaction parameters allow transactions to remain open for up to 30 days without lease renewals.
+
+```shell
+cd
+git clone https://github.com/pravega/pravega
+cd pravega
+git checkout r0.9
+./gradlew startStandalone \
+    -Dcontroller.transaction.lease.count.max=2592000000 \
+    -Dcontroller.transaction.execution.timeBound.days=30
+```
+
+### Install Apache Spark
+
+This will install a development instance of Spark locally.
```

Comment:
Add a sentence like `For example, we take Spark 3.0.2 here.` is a bit better, or we need to change these things every spark releases.

Reply:
Done.